src/feedback/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+import styled from "styled-components";
+import Alert, {
+  alertKindValues,
+  alertSeverityValues,
+} from "src/feedback/Alert/Alert";
+import { heading3, heading6 } from "src/guidelines/theme";
+
+const meta: Meta<typeof Alert> = {
+  component: Alert,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Alert>;
+
+const Title = styled.h1`
```

Comment:
You don't have to create your title and title2. There are here : purrfect-ui/src/utils/StorybookComponents/Titles.tsx